### PR TITLE
Lon 569

### PR DESCRIPTION
**Pull request by turran**

### CODE REVIEW

**Comment by vmartos** [comment]
I think a rebase to master is needed, also there are no changes in OneplayLongomatch?

File: LongoMatch.Core/Interfaces/GUI/IPanel.cs [hunk 1]
```diff
@@ -23,7 +23,8 @@ namespace LongoMatch.Core.Interfaces.GUI
 	public interface IPanel
 	{
 		event BackEventHandle BackEvent;
-		
+		void onLoaded ();
+		void onUnloaded ();
```
Comment (ylatuya) [inline LongoMatch.Core/Interfaces/GUI/IPanel.cs]: Methods should be PascalCase and new inteface methods must be documented

**Comment by turran** [comment]
Ready to review again

File: LongoMatch.GUI/Gui/Panel/SportsTemplatesPanel.cs [hunk 2]
```diff
@@ -143,6 +143,16 @@ public SportsTemplatesPanel ()
...
+		{
+
+		}
+
+		public void OnUnloaded ()
+		{
+
+		}
+
 		protected override void OnDestroyed ()
```
Comment (ylatuya) [inline LongoMatch.GUI/Gui/Panel/SportsTemplatesPanel.cs]: These methods should be moved after the destructors

File: LongoMatch.GUI/Gui/MainWindow.cs [hunk 3]
```diff
@@ -104,6 +104,10 @@ public partial class MainWindow : Gtk.Window, IMainController
 			}
 		}
 
+		public UIManager GetUIManager () {
+			return UIManager;
+		}
```
Comment (ylatuya) [inline LongoMatch.GUI/Gui/MainWindow.cs]: This should be a property with a getter to follow the same style.

File: LongoMatch.GUI/Gui/Panel/SportsTemplatesPanel.cs (hunk 2 above)
Comment (ylatuya) [inline LongoMatch.GUI/Gui/Panel/SportsTemplatesPanel.cs]: I have edited https://fluendo.atlassian.net/wiki/display/TEAM/Coding+Style to make clear that dispose and destroy callbacks are also part of the "Destroy" section for ordering

File: LongoMatch.GUI/Gui/MainWindow.cs (hunk 3 above)
Comment (turran) [inline LongoMatch.GUI/Gui/MainWindow.cs]: Yes, but I cannot create a Property with the same name of a private member, so:
```
public UIManager UIManager () {
  return this.UIManager;
}
```
does not work, as UIManager is generated by the UI Designer. That's why I created GetUIManager(). What should I do here?

**Comment by turran** [comment]
Review again please :)

File: LongoMatch.GUI/Gui/MainWindow.cs (hunk 3 above)
Comment (ylatuya) [inline LongoMatch.GUI/Gui/MainWindow.cs]: I didn't notice that, all good then

**Comment by ylatuya** [comment]
:+1:

**Pull request merged**